systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -527,4 +538,135 @@ public static void waitForTopicWithPrefixDeletion(String namespaceName, String t
                 }
             });
     }
+
+    /**
+     * Executes a command in the specified broker pod for a given data directory path.
+     *
+     * @param testStorage   the test storage object containing cluster information.
+     * @param brokerPodName the name of the broker pod.
+     * @param volumeId      the volume ID for the data directory.
+     * @param brokerId      the broker ID for the data directory.
+     * @param command       the command to execute in the pod.
+     * @return the ExecResult of the command execution.
+     * @throws RuntimeException if the command execution fails.
+     */
+    private static ExecResult executeInBrokerPod(TestStorage testStorage, String brokerPodName, int volumeId, int brokerId, String command) {
+        final String dataDirPath = String.format("/var/lib/kafka/data-%d/kafka-log%d", volumeId, brokerId);
+        LOGGER.info("Executing command in broker {} volume {} at path {}: {}", brokerId, volumeId, dataDirPath, command);
+
+        ExecResult execResult = cmdKubeClient(testStorage.getNamespaceName())
+            .execInPodContainer(brokerPodName, "kafka", "bash", "-c", command);
+
+        if (execResult.returnCode() != 0) {
+            throw new RuntimeException("Failed to execute command in " + dataDirPath + " in pod " + brokerPodName + ": " + execResult.out() + " " + execResult.err());
```

Comment:
Because this should be common for all of the commands, the dataDirPath is not correct right?